meas_std_val: 0.05
u_std: [0.03, 0.03, 0.03, 0.03]
initial_range: [1.0, 1.0, 1.0, 0.25, 0.25, 0.25]
num_particles: 250
stale_filter_time: 3.0

labels:
- cargo_red
- cargo_blue

bounds:
  - [-20.0, 20.0]
  - [-20.0, 20.0]
  - [-0.117, 1.0]
  - [-10.0, 10.0]
  - [-10.0, 10.0]
  - [-10.0, 10.0]
